tg_bot.py

```diff
@@ -250,6 +263,27 @@ def print_unsubscribe_button(msg):
         reply_markup=InlineKeyboardMarkup(custom_keyboard, one_time_keyboard=True))
 
 
+def print_subscription_status(update, msg, context): 
+    # include the parameters
+    """
+    Prints current subscription status
+    """
+    user_id = str(update.effective_user.id)
+
+    # format date 
+    subscription_limit = scheduled_jobs['limit_subs'].date()
+    date_object = subscription_limit.strftime("%d-%m-%Y")
+
+    # check if exists a schedule job
+    if user_id in scheduled_jobs:
+        msg.reply_text('Current subscription details:\n\n - Department: %s \n\n - Type: %s \n\n - Until: %s \n' % (
+                context.user_data['buro'], context.user_data['termin_type'], date_object))
+
+    # regardless of checking, prints every second day the status of schedule job
```

Comment:
Seems like dead comment, remove.

Reply:
It is a future development idea, removed.